### DISCUSSION TITLE
Stupid question?

### DISCUSSION THREAD

**Discussion by Fail-Fast-V0** [dz0ny/klipper-prusa-mk3s#15] [Q&A] [answered]
What does the instruction "Copy https://github.com/dz0ny/klipper-prusa-mk3s/blob/main/printer.template.cfg to printer.cfg in your klipper config" mean? Does this mean that I need to rename _printer.template.cfg_ to _printer.cfg_?

I noticed that the command copies the mk3 configs to _~/klipper_config/config_ while klipper is looking for the config files in _~printer_data/config_

If I rename _printer.template.cfg_ to _printer.cfg_then I get this error. 
![image](https://user-images.githubusercontent.com/99149033/212594643-ac7c390e-f9b9-4979-9ca7-30337c739947.png)

**Comment by Fail-Fast-V0** [comment]
I think _klipper_config_ is no longer used in lieu of _printer_data_

**Comment by dz0ny** [comment, marked as answer]
This is because of the change in how Klipper and Mainsail organize data. You can put the repo to any path and include it from there. Strangely the Moonraker should have migrated and patched the config for you, I believe that failed.  See https://github.com/Arksine/moonraker/issues/516

**Comment by f1yn** [comment]
Would you consider (or letting someone else) updating the repo to reflect the updated paths? Bootstrapping a fresh install is a little more convoluted due to these loading path changes.

**Reply by dz0ny** [reply]
Repo is open for PRs :)

**Reply by f1yn** [reply]
[Here is a pull request :octocat:](https://github.com/dz0ny/klipper-prusa-mk3s/pull/19)

**Comment by confuzled-dev** [comment]
> What does the instruction "Copy https://github.com/dz0ny/klipper-prusa-mk3s/blob/main/printer.template.cfg to printer.cfg in your klipper config" mean? Does this mean that I need to rename _printer.template.cfg_ to _printer.cfg_?
> 
> I noticed that the command copies the mk3 configs to _~/klipper_config/config_ while klipper is looking for the config files in _~printer_data/config_
> 
> If I rename _printer.template.cfg_ to _printer.cfg_then I get this error. ![image](https://user-images.githubusercontent.com/99149033/212594643-ac7c390e-f9b9-4979-9ca7-30337c739947.png)

To be clear, you are intended to copy the contents of the example_printer.cfg to the printer.cfg file. This is where you will make all (most) configuration changes

**Reply by f1yn** [reply]
Sorry, I replied to this while tired and thought your quote a new post.

**Comment by charminULTRA** [comment]
I'd also like to see the instructions updated for this line, it's vague and I don't know what to do.
